src/expression.c

```diff
@@ -3598,6 +3605,12 @@ Expression *ThisExp::toLvalue(Scope *sc, Expression *e)
     return this;
 }
 
+Expression *ThisExp::toLvalue14283(Scope *sc, Expression *e)
```

Comment:
I see no point in making this a separate function. Just inline it with its comment.